docs/reference/how-to/size-your-shards.asciidoc

```diff
@@ -221,25 +221,40 @@ GET _cat/shards?v=true
 // TEST[setup:my_index]
 
 [discrete]
-[[field-count-recommendation]]
-==== Data nodes should have at least 1kB of heap per field per index, plus overheads
+[[fields-heap-recommendation]]
+==== The heap of data nodes should suffice for the fields of indices, plus overheads
 
-The exact resource usage of each mapped field depends on its type, but a rule
-of thumb is to allow for approximately 1kB of heap overhead per mapped field
-per index held by each data node. In a running cluster, you can also consult the
-<<cluster-nodes-stats,Nodes stats API>>'s `mappings` indices statistic, which
-reports the number of field mappings and an estimation of their heap overhead.
+You can use the <<cluster-nodes-stats,Nodes stats API>>'s `mappings` indices
+statistic, to get the number of field mappings of indices and an estimation of
+their heap overhead per node.
+
+[source,console]
+----
+GET _nodes/stats/indices?human&filter_path=nodes.*.indices.mappings.total_estimated_overhead*
```

Comment:
On reflection I think it would be better to get all the data in a single call (plus node names too):

`/_nodes/stats?human&filter_path=nodes.*.name,nodes.*.indices.mappings.total_estimated_overhead*,nodes.*.jvm.mem.heap_max*`

It'd be useful to have some example output in these docs too.